charter.md

```diff
@@ -0,0 +1,61 @@
+# CNCF Public Sector User Group Charter
+
+Primary Authors: @onlydole
+
+Approved < Date TBD >
+
+Reviewed and contributed to by:
+
+- @idunbarh
+
+## Meetings
+
+- Biweekly on Thursdays @ 10am Pacific
+
+## Introduction
+
+This charter describes the operations of the CNCF Public Sector User Group, which focuses on developing and sharing cloud native practices for public sector organizations that are building and operating cloud native infrastructure or applications.
+
+This user group (UG) produces supporting material and best practices for end users in the public sector and provides guidance and coordination for CNCF projects working within the group's scope.
+
+## Mission
+
+The mission of the CNCF Public Sector User Group is to:
+
+- Collaborate on areas related to developing, distributing, deploying, managing and operating secure cloud native workflows that can be used by organizations in the public sector vertical.
+Develop informational resources including guides, tutorials, and white papers to give the community an understanding of best practices, trade-offs, and value-adds.
+- Identify useful CNCF projects and contexts.
+- Share information about gaps and opportunities for improvement with the CNCF ecosystem.
+
+## Areas considered in Scope
+
+The CNCF Public Sector User Group focuses on the following topics of the lifecycle of cloud-native applications:
+
+- Isolated Environment Cloud Native Computing
+- Far Edge Cloud Native Computing
```

Comment:
I recommend we follow these terms with a brief definition, e.g.

An isolated environment is an environment that, because of locality or security constraints, is never connected to a broader network (airgapped) and must therefore be provisioned and maintained through media transfer (sneakernet).

The far edge refers to systems that, because of size, weight, and power (SWAP) or locality constraints, is limited in scalability of computing resources and in connectivity to broader networks.

Reply:
I'm leaving data diode and country sovereign / classified cloud out of my definition of isolated because I think it's a separate concern, but we may want to add that to the list as well.

Reply:
In other venues, I've spoken with folks in the CNCF ecosystem also concerned with data sovereignty, so I do think mentioning it, as you said separately, is a good idea.